src/cohort-manager.js

```diff
@@ -1,9 +1,12 @@
 import Cohort from './cohort.js'
-import { Student, countID } from './student.js'
+import { Student } from './student.js'
+
+let countID = 1
 
 class CohortManager {
   constructor() {
     this.cohortList = []
+    this.allStudents = []
```

Comment:
I like how you've handled having a separate list of all students and assigning each of them an individual id. 

Do you reckon countID could be a property of CohortManager? I don't think it is very likely we'd have more than one instance of said manager, but if we did, would you want each to have their own id counter, or have one shared and updated by both?

Reply:
I mainly left it out for ease of exporting it. Also if another branch were to open up you could have another instance of cohort manager yet still have individual ids for all the companies students.

Reply:
fair point - had not thought of it that way!